### PR TITLE
feat: add sign in and link wallet modals

### PR DESCRIPTION
**Pull request by 6heLL66**

### CODE REVIEW

File: src/shared/components/copy_tag/ui.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          onClick={handleClick}
+          padding='4px 8px 4px 12px'
+          sx={{ width: 'fit-content' }}
+          background={theme.custom.colors.surface2.enable}
+          hover={theme.custom.colors.surface2.hover}
+          borderRadius='10px'
+        >
+          <Typography variant='caption2'>{`${text.slice(
+            0,
+            maxSymbols - END_SYMBOLS_COUNT,
```
Comment (ordinaryask) [inline src/shared/components/copy_tag/ui.tsx]: we have a function for that
"truncate" in shared/utils

File: src/widgets/modals/signIn/ui.tsx [hunk 2]
```diff
@@ -1,4 +1,6 @@
-import { Dialog, Typography } from '@mui/material'
+import { Box, Button, Dialog, Typography, useTheme } from '@mui/material'
+import { BorderButton } from 'shared/components/border_button'
```
Comment (ordinaryask) [inline src/widgets/modals/signIn/ui.tsx]: pls add index.ts file for shared/components

**Review by ordinaryask — CHANGES_REQUESTED** [review]

File: src/shared/components/copy_tag/ui.tsx (hunk 1 above)
Comment (ordinaryask) [reply src/shared/components/copy_tag/ui.tsx]: rename component file to camelCase, copyTag

Comment (6heLL66) [reply src/shared/components/copy_tag/ui.tsx]: всм
схуяли

**Review by ordinaryask — APPROVED** [review]

**Pull request merged**